cmd/kube-proxy/app/server.go

```diff
@@ -88,7 +95,8 @@ func checkKnownProxyMode(proxyMode string) bool {
 // Options contains everything necessary to create and run a proxy server.
 type Options struct {
 	// ConfigFile is the location of the proxy server's configuration file.
-	ConfigFile string
+	ConfigFile  string
+	WriteConfig string
```

Comment:
`--write-config-to` or `--write-config-path` would show that it is not a bool.

Actually, I was talking more about the Go field name in my comment above.